lib/ansible/modules/windows/win_service.py

```diff
@@ -113,6 +113,20 @@
     - A newly created service will default to C(LocalSystem).
     type: str
     version_added: '2.3'
+  recovery_reset_interval:
+    description:
+      - Reset failure counter after this number of seconds.
+    type: int
+    default: 0
+    version_added: '2.10'
+  recovery_actions:
+    description:
+      - A list representing the recovery actions you want to set (3).
```

Comment:
I would omit the 3, it seems like in the internal API you can set as many recovery actions as you want. It's just the GUI that limts to 3.